src/main/java/core/basesyntax/createproductslist/ProductsListImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.createproductslist;
+
+import core.basesyntax.operationswithfile.Operation;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ProductsListImpl implements ProductsList {
+    @Override
+    public HashMap getProductList(ArrayList<Operation> arrayList) {
```

Comment:
In my opinion we should return Map if we want follow SOLID principles

Reply:
add